vrk-kpa/suomidigi
# SUOPA-841 Content pages

To test: 
- `make fresh drush-cr`
- Open proto and search for article, core content and page. https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/
- Open https://suomidigi.docker.sh and search for article, core content and page. Compare the layouts and check the mobile layouts as well.
